src/components/Calculator.js

```diff
@@ -1,12 +1,47 @@
 import React from 'react';
```

Comment:
- [ ] You don't have a descriptive readme file in your repository :fearful: It would be very great if you add microverse readme file using this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) :+1:
- [ ] Also, don't forget to insert your own data after adding that readme template in your repo because most of the students forget about this after adding the readme template in their project :smile:

Reply:
I forget to merge the past activity, now is updated.
Thank you